it/it-tests/src/test/resources/organization/IssueAssignTest/one-issue-per-file-profile.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?><!-- Generated by Sonar -->
```

Comment:
We're doing this for every ITs to be isolated and to not be impacted if a profile is changed